src/zope/testrunner/tests/testrunner-coverage.rst

```diff
@@ -1,10 +1,21 @@
-Code Coverage
-=============
+===============
+ Code Coverage
+===============
+
+.. caution:: The coverage reports described here are in a format specific to
+			 zope.testrunner. They are not compatible with those
+			 produced by `coverage <https://coverage.readthedocs.io>`_.
```

Comment:
This makes it sound like zope.testrunner invented its own format.  That's not the case: we're using Python stdlib's [`trace.py`](https://docs.python.org/3/library/trace.html).

Reply:
Good point, I'll clarify.